css-two-column-layout/styles.css

```diff
@@ -0,0 +1,73 @@
+* {
+  font-family: Karla, sans-serif;
+  box-sizing: border-box;
+}
+
+p {
+  line-height: 2em;
+  color: grey;
+}
+
+h1 {
+  font-size: 60px;
+  margin-bottom: 0;
+}
+
+h3 {
+  width: 75%;
+}
+
+h1,
+h3,
+button {
+  text-transform: uppercase;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.row {
+  margin-top: 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.column-half {
+  display: flex;
+  width: 40%;
+}
+
+.column-one-thirds {
+  width: calc(100% / 3);
+}
+
+.column-two-thirds {
+  padding-top: 4%;
+  width: calc(200% / 3);
+}
+
+.picture {
+  width: auto;
+  max-height: auto;
+}
+
+.column-full {
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  width: 60%;
```

Comment:
always make sure that the column size matches the column name, if you want to the content to be 60% of the column, you can have the column at 100% width for a column full, and then adjust the width of the child element.

Reply:
Thanks, the edit worked out nicely